src/main/java/mate.academy/Main.java

```diff
@@ -10,7 +10,7 @@
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductService productService = (ProductService) injector.getInstance(ProductService.class);
```

Comment:
Make sure that the Injector class follows the checklist. It should have checks for annotations and an exceptions handling strategy as mentioned in the checklist.